cmd/download.go

```diff
@@ -176,6 +139,161 @@ func runDownload(cfg config.Config, flags *pflag.FlagSet, args []string) error {
 	return nil
 }
 
+type download struct {
+	// either/or
+	slug, uuid string
+
+	// user config
+	token, apibaseurl, workspace string
```

Comment:
I'm not sure if most would agree to save these an individual fields rather than the config object. Calling `GetString` throughout the system seems vulnerable prone to typo's where using fields has the benefit of compiler catching it.